apps/converter/Components/Area.js

```diff
@@ -0,0 +1,248 @@
+import React  from "zarkit/react";
+function AreaFunc(val,madeSelection_1,madeSelection_2) {
+    var result,formulvalue;
+    if (madeSelection_1 === madeSelection_2 ) {
```

Comment:
A lot of these `else if` statements can be optimized and generated through an object.
I recommend define a structure that cover your requirements for that and use it there kind of purposes.